index.js

```diff
@@ -66,11 +66,14 @@ var getConfig = function(lineString, lineCount) {
 					return typeof val === 'number' && !isNaN(val);
 				})
 			;
-
+			
 			if (range[0] > current + padding) {
-				collapse.push(current + '-' + (range[0] - 1 - padding));
+				var collapseEnd = (range[0] - 1 - padding);
+				if (collapseEnd !== current) {
```

Comment:
@cherifGsoul can you explain this logic change? Would this show the lines between a set of highlights like:

```
@highlight 1-3,33-55
```

`range[0] > current + padding` would be true when `range[0]` is `33`

Reply:
@justinbmeyer The logic is to compare the first line and the last line of collapsible:
- Using `current` which is initialized to `1` or calculated in this expression at the end of the loop `current = (range[1] || range[0]) + padding + 1`
- In each iteration of the loop `collapseEnd` is calculated based on the first item `range[0]` of the new range of line highlight `collapseEnd = (range[0] - 1 - padding)`

In short:
`current = currentRange[1] + 1 + padding`
`next = nextRange[0] - 1 - padding`

`next - current` should be greater than `1` to make a collapsible

So for `@highlight 1-3,33-55` we have a collapsible between `[7, 29]` and it shows only the 3 lines for padding between highlights:
`current = 3 + 1 + 3 // => start collapsible from line 7`
`next = 33 - 1 - 3 // => close collapsing at the line 29`